src/audio/google/google_rtc_audio_processing.c

```diff
@@ -497,6 +513,109 @@ static int google_rtc_audio_processing_trigger(struct comp_dev *dev, int cmd)
 	return comp_set_state(dev, cmd);
 }
 
+#if CONFIG_FORMAT_S16LE
+static void render_s16(struct google_rtc_audio_processing_comp_data *cd)
+{
+	struct comp_buffer *buffer_c = buffer_acquire(cd->aec_reference);
```

Comment:
Buffer_acquire() removal applies also for this usage, right?

Reply:
yep, we can remove after if this merges first.